### ISSUE TITLE
[SUGGESTION]: New Pokedex filter

### ISSUE THREAD

**Issue by The-Evil-Canadian** [pokeclicker/pokeclicker#4239]
### Current Behavior

Checking your Pokemon's EVs, shiny status, Pokerus status, etc. could be easier.

### Improved Behavior

If possible, I'd love to see a new toggle in the Pokedex to view only the Pokemon available in the current route/dungeon.

### Reason for change

Some Pokemon on routes and in dungeons are from different Generations and of different types, and manually searching can get annoying. Having the toggle in the Dex would eliminate these problems.

### Detailed Description

To use route 205 in Sinnoh as an example, it has Pokemon from all 4 Generations up to that point and of several different types. Sure we could just search the Pokemon up individually or use the Region and/or type filters, but having this toggle would make the whole thing easier. You could have everything set to "all" and go from there viewing only the Pokemon on that route! When you change routes or dungeons, loading up the Dex would then of course show you the ones in that new place.

**Comment by Farboo171** [comment]
Would this be different in a meaningful way from the wiki?

**Comment by The-Evil-Canadian** [comment]
Well, we wouldn't need to use the wiki if it were right there in the game. You also can't use the wiki to check your Pokemon's status and such, which is why I feel this is a great idea.

**Comment by RegisCoaxans** [comment]
#4004 closes this.

**Comment by The-Evil-Canadian** [comment]
Not really. Having all the route Pokemon displayed in the Dex would be a big help that #4004 doesn't help with
